Calculations.py

```diff
@@ -11,57 +11,51 @@ def math_model(all_characters, max_stats, promotion_gains, base_classes, promote
 	'''
 	
 	expected_stats = []
+	index = -1
+	count = 0
+	for character in all_characters:
```

Comment:
Is it necessary for this for loop to run until completion. Wouldn't it make sense to add a break ounce the character is found?